crates/interledger-packet/src/packet.rs

```diff
@@ -928,15 +955,29 @@ mod test_reject {
             let mut buffer = BytesMut::from(REJECT_BYTES);
             buffer.extend_from_slice(&[0x11, 0x12, 0x13]);
             buffer
-        })
-        .unwrap();
-        assert_eq!(with_junk_data.code(), REJECT_BUILDER.code);
-        assert_eq!(with_junk_data.message(), REJECT_BUILDER.message);
-        assert_eq!(
-            with_junk_data.triggered_by().as_ref(),
-            REJECT_BUILDER.triggered_by
-        );
-        assert_eq!(with_junk_data.data(), fixtures::DATA);
+        });
+
+        // feature = "strict" is used when fuzzing and is tested here to ensure
+        // error is returned instead of roundtripping when junk data is added
+        #[cfg(feature = "strict")]
+        {
+            assert_eq!(
+                "Invalid Packet: Packet contains unaccounted for trailing bytes",
+                format!("{}", with_junk_data.unwrap_err())
+            );
+        }
+
+        #[cfg(not(feature = "strict"))]
+        {
+            let with_junk_data = with_junk_data.unwrap();
+            assert_eq!(with_junk_data.unwrap().code(), REJECT_BUILDER.code);
```

Comment:
```suggestion
            assert_eq!(with_junk_data.code(), REJECT_BUILDER.code);
```

Extra unwrap left in. CI doesn't catch this because it doesn't try checking all crates with the default features. These tests probably only go through the `cargo test --workspace --all-targets`.